app/Http/Controllers/LandingPagesController.php

```diff
@@ -368,6 +368,16 @@ private function getLightbox($lightbox)
         ];
     }
 
+    public function getEditorialHeader()
+    {
+        $hour = Hour::today()->first();
+
+        $data['date'] = Carbon::now()->format('M d, Y');
+        $data['hours'] = $hour->present()->getTodayStatusWithHours();
+
+        return response()->json($data);
+    }
+
```

Comment:
This looks like duped code from the new FrontendController method. I think this can be deleted.